app/models/Item.scala

```diff
@@ -29,6 +29,20 @@ case class Item(id: Int,            // DB key
                 updated: Date,      // Time of last transfer
                 transfers: Int)  {  // Number of transfers
 
+  def collection = {
+    DB.withConnection { implicit c =>
+      SQL("select * from collection where id = {collection_id}")
+      .on('collection_id -> collectionId).as(Collection.coll.single)
+    }
+  }
+
+  def holds = {
```

Comment:
maybe `holdCount` would be better? `holds` makes me expect a list of Holds... we also use `xxxCount` in other models (see Publisher) for this type of method.